docs/framework/migration-guide/retargeting/4.0-4.5.1.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Retargeting Changes for Migration from .NET Framework 4.0 to 4.5.1 | Microsoft Docs"
+ms.custom: ""
+ms.date: "05/01/2017"
+ms.prod: ".net-framework-4.6"
+ms.reviewer: ""
+ms.suite: ""
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+ms.assetid: 0498da14-6aae-439f-97c7-6a6f609b86cf
+caps.latest.revision: 1
+author: "conniey"
+ms.author: "conniey"
+manager: "wpickett"
+---
+
+# Retargeting Changes for Migration from .NET Framework 4.0 to 4.5.1
+
+[!include[introduction](./introduction.md)]
+
+If you are migrating from the .NET Framework 4.0 to 4.5.1, review the following topics for application compatibility issues that may affect your app:
+
+## ADO.NET
+
+[!include[DbParameter.Precision and DbParameter.Scale are now public virtual members](./dbparameterprecision_and_dbparameterscale_are_now_public_virtual_members.md)]
+
+## ASP.NET
+
+[!include[MachineKey.Encode and MachineKey.Decode methods are now obsolete](./machinekeyencode_and_machinekeydecode_methods_are_now_obsolete.md)]
+
+[!include[Multi-line ASP.Net TextBox spacing changed when using AntiXSSEncoder](./multi-line_aspnet_textbox_spacing_changed_when_using_antixssencoder.md)]
+
+[!include[WebUtility.HtmlEncode and WebUtility.HtmlDecode round-trip BMP correctly](./webutilityhtmlencode_and_webutilityhtmldecode_round-trip_bmp_correctly.md)]
+
+## C\#
+
+[!include[Foreach iterator variable is now scoped within the iteration, so closure capturing semantics are different (in C#5)](./foreach_iterator_variable_is_now_scoped_within_the_iteration_so_closure_capturing_semantics_are_different_in_c5.md)]
+
+## ClickOnce
+
+[!include[Apps published with ClickOnce that use a SHA-256 code-signing certificate may fail on Windows 2003](./apps_published_with_clickonce_that_use_a_sha-256_code-signing_certificate_may_fail_on_windows_2003.md)]
+
+## Core
+
+[!include[Foreach iterator variable is now scoped within the iteration, so closure capturing semantics are different (in C#5)](./foreach_iterator_variable_is_now_scoped_within_the_iteration_so_closure_capturing_semantics_are_different_in_c5.md)]
```

Comment:
In the original Breaking changes document, it is listed under both categories. I can remove one if you want. https://github.com/Microsoft/dotnet-apiport/blob/master/docs/BreakingChanges/100%20Foreach%20iterator%20variable%20is%20now%20scoped%20within%20the%20iteration.md